src/aria/widgets/form/CheckBox.js

```diff
@@ -66,14 +66,6 @@ module.exports = Aria.classDefinition({
          * @type Boolean
          */
         this._mousePressed = false;
```

Comment:
You could completely remove `this._mousePressed` as it is not used anymore anywhere.